client/themes/morning.css

```diff
@@ -207,6 +207,17 @@ body {
 	color: #ccc;
 }
 
+#chat .join .content,
+#chat .kick .content,
+#chat .mode .content,
+#chat .nick .content,
+#chat .part .content,
+#chat .quit .content,
+#chat .topic .content,
+#chat .topic_set_by .content {
+	color: #999;
+}
```

Comment:
For future readers, this "resets" colors. There is effectively no change on this theme between master and this PR, but not adding this would make them much darker.